internal/pkg/mqtt/client_options.go

```diff
@@ -53,22 +42,32 @@ type ConnectionOptions struct {
 
 // MQTTClientOptions contains the client options which are loaded via reflection
 type MQTTClientOptions struct {
-	Username          string
-	Password          string
-	ClientId          string
+	// Client Identifiers
+	Username string
+	Password string
+	ClientId string
+	// Connection information
 	Topic             string
```

Comment:
Topic not needed for connection. Doesn't belong in Optional. Client may use one or many topics for publish or for subscribe.